Makefile

```diff
@@ -144,7 +144,11 @@ sure: checkformatting backend-lint frontend-lint backend-tests frontend-tests fu
 
 .PHONY: frontend-tests
 frontend-tests: node_modules/.uptodate
-	@$(GULP) test
+ifdef ARGS
+	yarn test $(ARGS)
+else
+	yarn test
+endif
```

Comment:
I'm ambivalent about this. On the one hand it ensures that `yarn test` and `make frontend-tests` are consistent. On the other hand its slightly slower than running `gulp test` as it introduces an extra process. There is also that issue with SIGINT handling and running `yarn test` vs `gulp test` that we ran into in the client recently.

Reply:
I am ambivalent too. The client uses `yarn test`.